### DISCUSSION TITLE
Assign to agent that initiate chat automatically

### DISCUSSION THREAD

**Discussion by wagkita** [chatwoot/chatwoot#10057] [Feature Requests]
When unassigned conversation is created, it's commom agent ignore the alert to assign chat. So, the chat is finished without agent.

My sugestion is when agent send any first message on unassigned chat, chatwoot automatically assign to agent sent the message.

This behavior might be a parameter.

**Comment by guidetti-ux** [comment]
Yes, a need with too, very import!!!

**Comment by diegoirs23** [comment]
Implementem isso, faz muito sentido.

**Comment by masterdaweb** [comment]
I wish this was implemented.
